lib/modules/helpers.js

```diff
@@ -0,0 +1,9 @@
+function getProfileLink(url) {
+  if (!url.startsWith("http")) return `http://${url}`
```

Comment:
Better to have `https` than `http`

Reply:
@Roshanjossey we can simply do that but not all links setup to work on `https`

Reply:
With browsers like Google Chrome and Brave, it'll automatically switch to `https` but in my experience, Firefox or Chromium doesn't do that.

Reply:
@Roshanjossey still some site only use `http` though they need to setup redirect so using `http` will support all